### DISCUSSION TITLE
Certificate verification bypass flag in interactsh-client

### DISCUSSION THREAD

**Discussion by hahwul** [projectdiscovery/interactsh#616] [Ideas]
Hi ProjectDiscovery. 
First of all, I'm always grateful for making good tools.

How about adding a flag (e.g. -unsafe-cert) to the interactsh-client for certificate verification bypass? Sometimes, it's challenging to renew the interactsh-server's certificate, and there are situations where using self-signed certificates becomes necessary. Providing this functionality as an optional feature could be helpful. I was thinking of writing the code and submitting a PR, but I wanted to ask for your opinion first.

**Comment by MetzinAround** [comment]
Thanks! 

https://github.com/projectdiscovery?view_as=public#work-with-us

We're always welcoming contributions, and I'd say go ahead and write it, but I'll defer to @ehsandeep about this feature idea.

**Reply by ehsandeep** [reply]
@hahwul Thank you, and yes, please go ahead with the PR.

**Reply by hahwul** [reply]
@ehsandeep @MetzinAround 
Sure! Thank you so much for all your input.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
